Makefile

```diff
@@ -258,7 +258,7 @@ bootstrap-only: lerna-bootstrap
 
 yarn-install: clean-all
 	# Install dependencies in individual packages so that we can link them at the top level.
-	for package in eslint/*/; do yarn --production --cwd $$package; done
+	for package in eslint/*/; do yarn --cwd $$package; done
```

Comment:
what's the difference between $$package and $package? (just looking at the other files)

Reply:
I think this is how `$` should be escaped in Makefile.

Reply:
That's right! I had to look this up the first time I edited this file haha